engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaUniversalJsonFunctionalTest.scala

```diff
@@ -266,7 +271,7 @@ class LiteKafkaUniversalJsonFunctionalTest extends AnyFunSuite with Matchers wit
         schemaMapAny,
         createObjectSchemaWithPatternProperties(Map("foo_int" -> schemaInteger)),
         validationMode = Some(ValidationMode.lax)
-      ), s"#/foo_int: expected type: Integer, found: String"),
```

Comment:
But right now we can't see where exactly the mistake is - I mean there is a missing field name.

Reply:
The original error comes from Everit validation. Now we catch such cases while encoding the value as JSON.

Reply:
Nice, thanks!